src/jesse_lib.erl

```diff
@@ -114,20 +115,20 @@ is_json_object_empty(_) ->
 %% lists with the same elements but in different order.  Lists for
 %% which order is relevant, e.g. JSON arrays, keep their original
 %% order and will be considered diffferent if the order is different.
--spec normalize_and_sort(Value :: any()) -> any().
-normalize_and_sort(Value) ->
-  normalize_and_sort_check_object(Value).
+-spec normalize_and_sort(Value :: any(), DraftVersion :: 3 | 4) -> any().
```

Comment:
what about drafts 1, 2 and 5+?

Reply:
Only draft 3 and draft 4 are supported by Jesse and Jessy.  I have by the way noticed that the default version is different, 4 in Jessy and 3 in Jesse.  I might be worthwhile to take that into account if we adopt upstream project.